packages/server/src/integrations/tests/sql.spec.ts

```diff
@@ -342,25 +342,6 @@ describe("SQL query builder", () => {
     })
   })
 
-  it("should use greater than when only low range specified", () => {
```

Comment:
Was this duplicated?

Reply:
Yes, there is a lint that fires on identical strings in `it` and `describe` blocks and there is an identical test elsewhere in this file.